agent.go

```diff
@@ -307,7 +292,7 @@ func (c *controller) agentInit(listenAddr, bindAddrOrInterface, advertiseAddr, d
 		return err
 	}
 
-	ch, cancel := nDB.Watch(libnetworkEPTable, "", "")
```

Comment:
There was actually a bug here.
At line 319 the ep table cancel function was expected:

`epTblCancel:       cancel,`